src/Symfony/Component/Notifier/Bridge/Firebase/FirebaseOptions.php

```diff
@@ -22,12 +22,10 @@
  */
 abstract class FirebaseOptions implements MessageOptionsInterface
 {
-    /** @var string the recipient */
+    /** the recipient */
```

Comment:
I would remove that doc block completely, tbh.

Reply:
Agreed